nltk/tokenize/moses.py

```diff
@@ -236,7 +236,7 @@ class MosesTokenizer(TokenizerI):
                                 ESCAPE_LEFT_SQUARE_BRACKET,
                                 ESCAPE_RIGHT_SQUARE_BRACKET]
 
-    def __init__(self, lang='en'):
+    def __init__(self, lang='en', no_escaping=False):
```

Comment:
Actually, I think the `escape` parameter should go to the `tokenize()` function instead of the initialization.